admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -249,6 +250,19 @@
 
         this.applyIframeStyling();
 
+        // if form is not found it means that combination is missing
+        const editionForm = iframeBody.querySelector<HTMLFormElement>(ProductMap.combinations.editionForm);
```

Comment:
Since it is only used one time, maybe we can skip the const and directly test `if (!iframeBody.querySelector<HTMLFormElement>(ProductMap.combinations.editionForm)`

Reply:
maybe you are right, but Neo already approved, so I think its not worth blocking PR for this little thing :wink:

Reply:
Yh sure. free comment ;)